ee_tests/src/page_objects/main_dashboard.page.ts

```diff
@@ -184,6 +185,23 @@ Page layout
     return spaceDashboard;
   }
 
+  // tslint:disable-next-line:max-line-length
+  async createSpaceWithNewCodebase(spaceName: string, templateName: string, strategy: string) {
+    await this.header.recentItemsDropdown.createSpaceItem.select();
+
+    support.info('Creating space');
+    await this.newSpaceName.enterText(spaceName);
+    await this.devProcessPulldown.enterText('Scenario Driven Planning');
```

Comment:
This should be `Scenario Driven Development`, search whole codebase for this wrong string